src/gui/AstroCalcDialog.cpp

```diff
@@ -256,6 +259,8 @@ void AstroCalcDialog::createDialogContent()
 	ui->phenomenToDateEdit->setDateTime(currentDT.addMonths(1));
 	ui->transitFromDateEdit->setDateTime(currentDT);
 	ui->transitToDateEdit->setDateTime(currentDT.addMonths(1));
+	ui->lunareclipseFromDateEdit->setDateTime(currentDT);
```

Comment:
Maybe change default values for both fields will be good idea: the first field should contains first day of current year and second field - the last day of current year. This is cosmetics.

@worachate001 @gzotti how about this idea?

Reply:
Yes, indeed. A default of "search all eclipses of this year" sounds reasonable. Alternatively, from "today" to "1 year in the future".